text/0000-cloud-based-mapping.md

```diff
@@ -0,0 +1,134 @@
+# Cloud-Based Mapping
+
+## Summary
+[summary]: #summary
+
+The goal of this RFC is to describe an approach to build out the Cartographer system into a collaborative, cloud based mapping solution, i.e. we want to support a fleet of mapping agents running Cartographer in pure localization mode against a map while forwarding the required information to formulate the global SLAM problem in the cloud.
+A central cloud instance receives local SLAM results and assembles them into a global map of the environment combining information across time and agents.
+By fanning out global map updates to agents, individual agents can benefit from world observations made by other agents with the cloud acting as an arbiter.
+
+## Motivation
+[motivation]: #motivation
+
+In many robotics applications multiple robots simultaneously map and localize within a continuously changing environment.
+
+Forwarding local SLAM results from agents to a central instance and solving the global SLAM problem in the cloud has the following advantages
+
+- by involving multiple robots sharing information (e.g. sensor measurements or intermediate mapping results) among one another, local world observations can be combined into a global and more detailed map of the world and the mapping process expedited.
+- by moving some of the computational load of mapping away from battery-powered agents to a central cloud instance with virtually unbounded computational resources, the mapping performance and runtime of agents can be enhanced with reduced hardware requirements.
+
+
+## Approach
```

Comment:
Not sure if it belongs here, but what about scaling? Cloud cartographer can't solve significantly larger problems than we currently solve on the robots without some changes. Should this be mentioned here?

Reply:
Probably something for the last section.